lib/tyxml/tyxml_js.ml

```diff
@@ -78,9 +75,14 @@ module Xml = struct
 
   let attach_attribs e l =
     List.iter (fun (n,att) ->
+      let n = Js.string n in
         match att with
-        | Attr a -> ignore(e##setAttributeNode(a))
-        | Event h -> Js.Unsafe.set e (Js.string n) (fun ev -> Js.bool (h ev))
+        | Attr a ->
+          let _ : unit React.S.t = React.S.map (function
```

Comment:
Assuming that GC of signals on javascript might be one day fixed, shouldn't this use `Lwt_react.S.keep` ?